lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -158,6 +158,27 @@ def run_commands(self, commands, check_rc=True):
         except ConnectionError as exc:
             self._module.fail_json(msg=to_text(exc))
 
+    def cli_run_commands(self, commands, check_rc=True):
```

Comment:
I don't like the method name, mostly because this doesn't have anything to do with cli. nxapi has the same problem with unconverted commands, and there's now a cli_run_commands method in Nxapi, which is confusing. Since all this does is wrap run_commands, I would rather see this in run_commands, possibly as a retry_json flag. (What I would probably prefer the most is the option of `check_rc='retry_json'`, since that path assumes check_rc is False, but that might be needlessly complicating the method signature)